File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--complex-data-binding-in-vsto-add-in-project_2.cs
            this.addressTableAdapter = new AdventureWorksLTDataSetTableAdapters.AddressTableAdapter();
            this.adventureWorksDataSet = new AdventureWorksLTDataSet();
            this.addressTableAdapter.Fill(this.adventureWorksDataSet.Address);
            this.addressBindingSource = new System.Windows.Forms.BindingSource();

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--complex-data-binding-in-vsto-add-in-project_2.vb
        Me.addressTableAdapter = New AdventureWorksLTDataSetTableAdapters.AddressTableAdapter()
        Me.adventureWorksDataSet = New AdventureWorksLTDataSet()
        Me.addressTableAdapter.Fill(Me.adventureWorksDataSet.Address)
        Me.addressBindingSource = New System.Windows.Forms.BindingSource()